typescript/infra/config/environments/testnet3/core.ts

```diff
@@ -0,0 +1,97 @@
+import { ChainMap, CoreConfig } from '@hyperlane-xyz/sdk';
+
+import { TestnetChains } from './chains';
+
+export const core: ChainMap<TestnetChains, CoreConfig> = {
+  alfajores: {
+    multisigIsm: {
+      validators: [
+        '0xe6072396568e73ce6803b12b7e04164e839f1e54',
+        '0x9f177f51289b22515f41f95872e1511391b8e105',
+        '0x15f77400845eb1c971ad08de050861d5508cad6c',
+      ],
+      threshold: 2,
+    },
+  },
+  fuji: {
+    multisigIsm: {
+      validators: [
+        '0x9fa19ead5ec76e437948b35e227511b106293c40',
+        '0x227e7d6507762ece0c94678f8c103eff9d682476',
+        '0x2379e43740e4aa4fde48cf4f00a3106df1d8420d',
+      ],
+      threshold: 2,
+    },
+  },
+  mumbai: {
+    multisigIsm: {
+      validators: [
+        '0x0a664ea799447da6b15645cf8b9e82072a68343f',
+        '0x6ae6f12929a960aba24ba74ea310e3d37d0ac045',
+        '0x51f70c047cd73bc7873273707501568857a619c4',
+      ],
+      threshold: 2,
+    },
+  },
+  bsctestnet: {
+    multisigIsm: {
+      validators: [
+        '0x23338c8714976dd4a57eaeff17cbd26d7e275c08',
+        '0x85a618d7450ebc37e0d682371f08dac94eec7a76',
+        '0x95b76562e4ba1791a27ba4236801271c9115b141',
+      ],
+      threshold: 2,
+    },
+  },
+  goerli: {
+    multisigIsm: {
+      validators: [
+        '0x95b76562e4ba1791a27ba4236801271c9115b141',
```

Comment:
This should be `0xf43fbd072fd38e1121d4b3b0b8a35116bbb01ea9`

from get-key-addresses:
```
  {
    "identifier": "alias/hyperlane-testnet3-key-bsctestnet-validator-2",
    "address": "0x95b76562e4ba1791a27ba4236801271c9115b141"
  },
  {
    "identifier": "alias/hyperlane-testnet3-key-goerli-validator-0",
    "address": "0xf43fbd072fd38e1121d4b3b0b8a35116bbb01ea9"
  },
```

Reply:
Good catch! It will be easier now that we have a single source of truth. Not sure what happened here...